code/modules/mining/lavaland/megafauna_loot.dm

```diff
@@ -728,6 +728,7 @@
 			consumer.eye_color_left = "#FEE5A3"
 			consumer.eye_color_right = "#FEE5A3"
 			consumer.set_species(/datum/species/lizard)
+			consumer.update_body()
```

Comment:
is this needed anymore?

Reply:
Yes? I moved eye refresh into update_body so you'll need to call it.

Reply:
set_species wouldn't call this for us?

Reply:
oh right